src/handleProviderRequest.ts

```diff
@@ -480,12 +479,17 @@ export const handleProviderRequest = ({
               // Generic error that will be hanlded correctly in the catch
               throw new Error('next');
             }
+
             // Let's try to fwd the request to the provider
             const provider = getProvider({
               chainId: activeSession?.chainId,
-            }) as StaticJsonRpcProvider;
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            response = await provider.send(method, params as any[]);
+            });
+
+            // Use Viem provider adapter as a fallback
+            response = await provider.request({
+              method,
+              params,
+            } as any); // TODO: Fix this but i cba right now
```

Comment:
i could not figure this type out but would love to not do this here if possible!